x-pack/plugins/index_management/common/constants/home_sections.ts

```diff
@@ -19,3 +22,16 @@ export enum IndexDetailsSection {
   Settings = 'settings',
   Stats = 'stats',
 }
+
+export type IndexDetailsTabIds = IndexDetailsSection | 'string';
```

Comment:
Is this meant to be `export type IndexDetailsTabIds = IndexDetailsSection | string;` so that we allow any string for the Id?

Reply:
Yes, the first type provide autocompletion support in the IDE for the "known" tabs like "Overview" or "Settings" and `string` is to allow any other dynamic ids for the tabs registered via the extension service.

Reply:
I didn't realised that you mean the literal `'string'` in the type declaration, that is a typo, fixed in the latest commit